syft/execution/computation.py

```diff
@@ -1,17 +1,22 @@
 import syft as sy
+from syft.common.util import TranslationTargets
 from syft.workers.abstract import AbstractWorker
 
 from syft.execution.action import Action
 from syft.execution.placeholder import PlaceHolder
 from syft.execution.placeholder_id import PlaceholderId
 
 from syft_proto.execution.v1.computation_action_pb2 import ComputationAction as ComputationActionPB
+from pythreepio.threepio import Threepio
+from pythreepio.utils import Command
+from pythreepio.errors import TranslationMissing
 
 
 class ComputationAction(Action):
     """Describes mathematical operations performed on tensors"""
 
-    def __init__(self, name, target, args_, kwargs_, return_ids):
+    def __init__(self, name, target, args_, kwargs_, return_ids, 
+                 base_framework=TranslationTargets.PYTORCH.value):
```

Comment:
Since we'll be translating full `Plans` and probably not inter-mingling actions targeted at different frameworks (yet anyway),  I imagined this kind of logic living in the `PlanTranslator` class. Want to keep the `Action` classes as close to pure data structures as possible.